backend/pyproject.toml

```diff
@@ -25,7 +25,7 @@ invenio-accounts = {git = "https://github.com/inveniosoftware/invenio-accounts.g
 invenio-app= "==1.2.3"
 invenio-base= "==1.2.0"
 invenio-cache= "==1.0.0"
-invenio-celery= "==1.1.0"
+invenio-celery= "==1.1.2"
```

Comment:
got a version conflict without this. @drjova is it the right solution?

Reply:
what was the conflict? ``v1.1.2`` seems safe (just pinning deps) but we haven't test it

Reply:
https://github.com/inveniosoftware/invenio-celery/commits/master